input.go

```diff
@@ -4,7 +4,9 @@ package githubv4
+
```

Comment:
This blank line looks like it shouldn't be there.

Reply:
Ack'd, I can fix that

Reply:
Fixed in c476f0c1d64e4255a76ab4467fd4ad6553475065.